<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="graphics.qrc">
    <normaloff>:/winicons/graphics/settings.png</normaloff>:/winicons/graphics/settings.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="creator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>ICVX Creator:</string>
    </property>
    <widget class="QLineEdit" name="pdf">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="avi">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf_btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QPushButton" name="avi_btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select PDF:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select AVI:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dict">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings:</string>
    </property>
    <widget class="QLineEdit" name="accept">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="accept_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="reject">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="reject_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Accepted dir:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Rejected dir:</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="inActive">
    <property name="geometry">
     <rect>
      <x>317</x>
      <y>120</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>In active directory</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="deleteRejected">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Delete rejected</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
